internal/services/formatters/hcl/checkov/formatter.go

```diff
@@ -75,16 +76,18 @@ func (f *Formatter) getDockerConfig(projectSubPath string) *dockerEntities.Analy
 
 func (f *Formatter) parseOutput(output string) error {
 	var vuln *entities.Vulnerability
-
 	binary, _ := ansi.Strip([]byte(output))
+	// For some reason checkov returns an empty list when no vulnerabilities are found
+	// and an object if vulnerabitilies are found, this checks ignores result when we have no vulnerabilities
+	if bytes.Equal(binary, []byte{91, 93, 13, 10}) {
```

Comment:
json.Unmarshal can handle empty list, so I think that this check is not necessary.

Reply:
Basically we have to types of return for checkov: empty list and object.
- Empty List returns occurs when no vulnerabilities are found.
- Object occurs when vulnerabilities are found.

We can't use `err:= json.Unmarshall(binary, &entities.Vulnerability)` to both cases since entities.Vulnarability must be object in one case and slice in other case. To avoid make more checks and more types i created this check.

Reply:
Ohhhh, I get it, make sense. So I think that is better change the `[]byte{91, 93, 13, 10}` to `[]byte("[]")` to more readability.